### PR TITLE
Add promo screenshots generation

### PR DESCRIPTION
**Pull request by loremattei**
This PR adds the scripts and the resources required to generate the enhanced screenshots for Android. 
It also moves all the screenshots related lanes to a new dedicated file.

### Download promo strings
This lane downloads the translations for the promo strings from GlotPress. 
1. Run `bundle exec fastlane download_promo_strings`
2. Verify that the new strings are downloaded 
_Note:_ It's possible that there's no new string at the moment, so you may want to delete the current ones from the `playstoreres/metadata/[langs]` before testing.

### Create promo screenshots
This lane creates the enhanced screenshots. It uses the screenshots in the `screenshots_orig` folder as a base. If the folder doesn't exists, the lane offers to create it and to populate with the images that it founds in the default location (output of the `screenshots` lane).

1. Prerequisites: you need to generate the standard screenshots with the `screenshots` lane.
2. Run `bundle exec fastlane android_promo_screenshot`
3. Verify that the script asks to generate the `screenshots_orig` folder. Answer `yes`.
4. Wait :-) 
5. Verify that the promo screenshots are generated.

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -2,32 +2,36 @@ default_platform(:android)
...
+  { glotpress: "de", google_play: "de-DE",  promo_config: {} },
+  { glotpress: "en-gb", google_play: "en-US",  promo_config: {} },
+  { glotpress: "es", google_play: "es-ES",  promo_config: {} },
+  { glotpress: "fr", google_play: "fr-CA",  promo_config: {} },
+  { glotpress: "fr", google_play: "fr-FR",  promo_config: {} },
+  { glotpress: "he", google_play: "iw-IL",  promo_config: {font: "/Library/Fonts/Arial Unicode.ttf", text_size: 60} },
+  { glotpress: "id", google_play: "id",  promo_config: {} },
+  { glotpress: "it", google_play: "it-IT",  promo_config: {} },
+  { glotpress: "ja", google_play: "ja-JP",  promo_config: {font: "/Library/Fonts/Arial Unicode.ttf", text_size: 60} },
+  { glotpress: "ko", google_play: "ko-KR",  promo_config: {font: "/Library/Fonts/Arial Unicode.ttf", text_size: 60} },
```
Comment (jtreanor) [inline fastlane/Fastfile]: Why is it required to configure fonts for some locales but not others?

Comment (loremattei) [reply fastlane/Fastfile]: There's a default font (Noto) defined in the plugin which is the one chosen by the designers for the screenshots. Unluckily, that font doesn't include a great support for not-latin characters, so we need to supply a different with better support for the locales that require them.

Comment (jtreanor) [reply fastlane/Fastfile]: Ah, that makes sense 👍

**Comment by loremattei** [comment]
@jkmassel I investigated the two issues you found.

> No such file or directory @ dir_s_mkdir - /Users/jeremymassel/Projects/WordPress-Android/fastlane/metadata/android//id/images/phoneScreenshots
> I received this error after a successful screenshot run. I'm not sure if this is something that'll be solved in the merge, but using this branch, fresh, to do a screenshot run then convert them, it happened fairly consistently.

I looked into it and it seems to be related to the basic screenshot generation: Java still uses some deprecated locales id and it creates a few issues. One of them is the _in_ => _id_ one. The screenshots are generated in the _in_ folder, but everything else expects it to be in the _id_ one (including Google Play). It seems that we have a similar issue with the strings as well, currently handled with a simlink. 

How we didn't spot it before, I don't know, but I guess it's also possible to something we or fastlane changed recently in the way we handle the locale changes.

I added a new lane that copies the screenshots from _in_ folder to _id_. It's a hack and I guess it's going to be quite fragile, since it depends on a lot of moving factor (android sdks, fastlane handling of locales, maybe our handling, etc.). Because of this, I made it an additional step in the lane instead of figuring out something more elegant inside the action, because I think that it'll be easier to fix if it'll break and/or to remove when the framework will handle it correctly.

> [15:11:30]: Missing gem 'fastlane-plugin-wpmreleasetoolkit', please add the following to your local Gemfile:
[15:11:30]:
[15:11:30]: ▸ gem "fastlane-plugin-wpmreleasetoolkit"
[15:11:30]:
[15:11:30]: Error loading plugin 'fastlane-plugin-wpmreleasetoolkit': Add 'gem "fastlane-plugin-wpmreleasetoolkit"' to your Gemfile and restart fastlane
[15:11:30]: It seems like you wanted to load some plugins, however they couldn't be loaded
[15:11:30]: Please follow the troubleshooting guide: https://docs.fastlane.tools/plugins/plugins-troubleshooting/

This one doesn't seem related to this PR at all. It doesn't happen to me and it seems that's a local error with fastlane that can't install the plugin.

**Comment by jkmassel** [comment]
Ran through the tests, end to end:

```
bundle exec fastlane android screenshots
bundle exec fastlane download_promo_strings
bundle exec fastlane android create_promo_screenshot
```

and everything worked perfectly!

**Pull request merged**